src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -191,9 +191,17 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
             // need to load the pending item
             this.fetchFeedItems();
         },
-        updateTask: ({ text, id }: { id: string, text: string }): void => {
```

Comment:
Is this still used by the "legacy" tasks? If so, does it still work properly for them? It looks like it was calling the other `updateTask` endpoint previously.

Reply:
No, this is nested under the newTaskAPI. This handler is only used if `newApi` feature is enabled